app/src/main/java/io/intrepid/contest/screens/sendinvitations/SendInvitationsActivity.java

```diff
@@ -48,16 +49,20 @@ public static Intent makeIntent(Context context) {
     }
 
     @Override
+    protected int getLayoutResourceId() {
+        return R.layout.activity_send_invitations;
+    }
+
+    @Override
     protected void onViewCreated(Bundle savedInstanceState) {
         super.onViewCreated(savedInstanceState);
 
-        setActionBarTitle(R.string.invite_contestants_bar_title);
         setActionBarDisplayHomeAsUpEnabled(true);
     }
 
     @Override
-    protected int getLayoutResourceId() {
-        return R.layout.activity_send_invitations;
+    public void setActionBarTitle(@StringRes int titleResource) {
```

Comment:
I actually do it so that my presenter can access this method as well (via `view.setActionBarTitle()`. Do you think it's bad practice that I created it with the super's method name in the Contract?